testdata/golang/light/jacobiSymbol.go

```diff
@@ -0,0 +1,33 @@
+package light
+
+import (
+	"fmt"
+)
+
+func jacobiSymbol(k, n uint64) (int, error) {
+	if k < 0 || n%2 == 0 {
+		return 0, fmt.Errorf("Invalid value. k = %d, n = %d", k, n)
```

Comment:
Can we handle `fmt.Errorf`?

Reply:
symflower? not sure... but also not the focus right now.